### DISCUSSION TITLE
How do I add a subheader and a divider to a `v-select` 's dropdown?

### DISCUSSION THREAD

**Discussion by josephlevarato** [vuetifyjs/vuetify#19019] [Q&A]
I'd like to use the subgroups inside the `VListItem` component from a `VSelect`, however I don't really understand how it works...

I tried to reuse the doc's example with [this playground](https://play.vuetifyjs.com/#eNqNUstqwzAQ/JWtelALSdw8C8Ep9NT2C3qIc3DtTSuQLSGtDcX43yv5AW4MiU/Szu6MZtAeK2ZNErxqvSgLZHsWEmZaxoQvUQ4QlvNY6+baFInKKRY5mg5qQIsSE4JynqkU5SFihJYiBnvhtKyrm9MBQS8UXCp5pH0oDAYGXGkTIzSBRSpaHyLTyhBUYPAMNZyNyoA779yPAzhdS+AdwMGPPHD+OOw0Xlzr2FqpgH417oHb4usH4xQNnwEJkh57M6rQcL/kUM+68T52P/Lh5PxE1wcoY1m4xrIDbhBXI+JqGnE9Iq4viX2yVJTC5xp3rmRe3cy8GTnYTLO+HRG304i7EXH3j3ga/nNmv/sFeEcpFXwqI9M7vwxh0C6VWydWz9h6sVk8s/ZcPrHTH2UM5Xo=), however as you can see I have a warning saying that `divider` is not a valid VNode.

What bothers me is that the doc doesn't say how to use `item-type`, and the prop is missing its description.

How does my code is supposed to look like?

Thank you in advance

**Comment by crystalfp** [comment]
Well, I implemented it without problems in: [Vuetify play](https://play.vuetifyjs.com/#eNqlVk2P2zYQ/SuM9iAZsKX9chY11ukWRdoE6KFogvaw3oNsjWymlKiQlBLD8H/PI0VpZXuRGMhBkDifb8g3Qz3uAl5UUplJkVbxJy3LYBbsFiVjC6/Qi2DGnMTKmprsehFsjKn0LElWWQm3jARvVFySScqqSB5glqi6NLygSSaLh5t4Gl9Pk4xrM5THpIvJUskvmhSiLILxIE8CYUNqoqjMSJE6N++R20HuI9138xueb8/I6S2R5uoqvmvTeNlEpEttc7jYNvR+Ue6DcSB4+b8+2u2Vdjv92CH4uaQumg311GbUapX8VlUxjJDy3lBRidTQG2ty30zSqnKfbrGSpUl5ScqLnFCToJVhzaSQwDJfBIa0WQRsxhFLY+3eECRdoOQ4kpW0ie6TAQAs9UrxyjBNpm5xtNRjO6YoZ3uWK1mwENhDa84Y4mrDLAI2tyZRGI6GGocFKr+ZO2a2Fc1YqOvlhlIcfThmhhthZX8qWVfs4ipke3/6nuysN3mPcNbC6xlrUlFDceUFP3C8PnG8Ps/x5sTx5tixqyzjDbd1nWq+U/P1D2u+PUFwex706Ynj9DzH1yeOrw8cwefncy70uiPAOxJCsv+kEtkrS4b7pCUV6OTpX4CJsdGgf8+ulSJwEH1xxLHewPdUr467Lns2st6d0nfYsxIyR2rH2xYzOgCY+8wRHsCFNIY+5+uYlJLqXVpmghQsic3fuHKloFYXkSO7dak1RR5RLyskBmwUXuDbboSv3sFoy1dUyIY+mK2gj+k6gs3fwM81xakQ0aOQafYHGjcajZn9fm+TR6MnlyCvy5XhssSmD4OwaNQeqO9MSOYsk6u6oNLEn2tS2w9ugAB9eNFNKrMh3APaxtAbImPhovdzFr1ChBFymFq548YyrlKMbfNWwAcx2/y/b7jIImuMMXeAzyJ36FxkFm3AEo+xDctK+sJ84VGkCPvb0BjKT4A5spvuGdqWZEf2sKb2AD2aKLTqFj5zpsAnYB4OihsoLRho7WsgTbPsbYNof2GaE4YmoqIItK0HNwx/auuo4YxdAa1tD3cps20MRuDii2wAp9+/uGv27Pvz9Ht1uJtR2F1OOWx1vJZyLSitOO4dWSS4e37N04KL7fwfuZRGzq4uL8c3eG7xTPHc4fnl8tJu2Gl+x7dzAbz8G1Bk3EF7uIu/WjjofUOKpyIjzdcltxniAuMAOg/Ct4mn5otz4t+jYfA8ChYlfXWmLVe6qdG1uXeMXEGO9P0fVUZ5WgvzsRWCRuuN6SZgkrhDwsoeVLAfB+2vDKC2934wzlOh6fB272QeRDDe9Z+zzm//9A2DVEsU). Your example has an old version of vue that prevents working.
